mt-kahypar/partition/refinement/rebalancing/advanced_rebalancer.cpp

```diff
@@ -28,13 +28,12 @@
 
 #include <array>
 #include <optional>
+#include <random>
 
 #include "mt-kahypar/partition/refinement/gains/gain_definitions.h"
 #include "mt-kahypar/utils/cast.h"
 #include "mt-kahypar/partition/context.h"
 
-#include "pcg_random.hpp"
-
```

Comment:
This means we can also remove pcg from `external_tools`, right? It would be good to also include this in the PR

Reply:
Done :)